src/evaluators.js

```diff
@@ -128,10 +111,27 @@ export function createSafeEvaluatorFactory(
         // Rewrite the source, threading through rewriter state as necessary.
         const rewriterState = allTransforms.reduce(
           (rs, transform) => (transform.rewrite ? transform.rewrite(rs) : rs),
-          { src }
+          { src, endowments }
         );
         src = rewriterState.src;
 
+        // todo (shim limitation): scan endowments, throw error if endowment
+        // overlaps with the const optimization (which would otherwise
+        // incorrectly shadow endowments), or if endowments includes 'eval'. Also
+        // prohibit accessor properties (to be able to consistently explain
```

Comment:
This todo comment is stale in light of https://github.com/Agoric/realms-shim/issues/36